### PR TITLE
#160583962 Users should be able to view and edit their profiles

### PR DESCRIPTION
**Pull request by nakimera**
####  What does this PR do?
This PR enables a user to view and edit their profile

####  Description of task to be completed
Users should be able to amend/edit their profiles depending on change of preference to the full disclosure of the individual user

####  How should this be manually tested?

- Clone this branch

`git clone origin ft-view-and-edit-profile-160583962`

- Install dependencies

`yarn install`

- Start server

`yarn start`

- Log into your Authors Haven account 

- Click view user profile button to view your profile
- Click edit user profile button to edit your profile
- Update your profile according to your preference

#### Screenshots

View user profile

screen shot 2018-11-13 at 17 03 26

Edit user profile

screen shot 2018-11-13 at 17 11 25

Updated user profile

screen shot 2018-11-13 at 17 06 54

####  Relevant Pivotal Tracker Stories
[#160583962](https://www.pivotaltracker.com/n/projects/2198584/stories/160583962)

### CODE REVIEW

**Comment by Edward-K1** [comment]
Good work @nakimera , however, the image upload failed. Did you forget to include any required information such as environment variables?
I also think this is a little too wordy:
> Users should be able to amend/edit their profiles  `depending on change of preference to the full disclosure` of the individual user

**Review by balaakagordon — APPROVED** [review]
I know it works because I tested it with you, however, as Edward has pointed out, there's something up with the image upload. It's taking a while to load the image browsing.

**Comment by balaakagordon** [comment]
Thanks for making the changes. Now it really LGTM

**Comment by nakimera** [comment]
@Edward-K1 sorry about that. I had to add the environment variables to Heroku. It should be working fine now.

**Review by kenneth051 — CHANGES_REQUESTED** [review]
LGTM. could you please fix the merge conflicts

**Comment by nakimera** [comment]
@kenneth051 The conflicts have been resolved

**Review by Edward-K1 — APPROVED** [review]
LGTM.

**Review by kenneth051 — APPROVED** [review]

**Comment by kenneth051** [comment]
LGTM
